lib/nylas/resources/providers.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require_relative "base_resource"
+require_relative "../handler/api_operations"
+
+module Nylas
+  # Providers
+  class Providers < BaseResource
+    include Operations::Get
+    include Operations::Post
+
+    def initialize(parent)
+      super("providers", parent)
+    end
+
+    def check_credentials
+      raise "client_id is required" if client_id.nil?
+    end
+
+    def list
+      check_credentials
+
+      get(
+        "#{host}/connect/providers/find",
+        query_params: { client_id: client_id },
+        api_key: api_key
```

Comment:
There's no `api_key` param defined it seems

Reply:
fixed